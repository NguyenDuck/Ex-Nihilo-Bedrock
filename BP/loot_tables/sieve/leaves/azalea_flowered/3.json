{
    "pools": [
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "loot_table",
                    "name": "loot_tables/leaves/golden/3.json"
                }
            ]
        }
    ]
}