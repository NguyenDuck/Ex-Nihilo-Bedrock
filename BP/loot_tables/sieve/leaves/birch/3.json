{
    "pools": [
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "loot_table",
                    "name": "loot_tables/sieve/leaves/birch/2.json"
                }
            ]
        },
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "item",
                    "name": "sapling",
                    "functions": [
                        {
                            "function": "set_data",
                            "data": 2
                        }
                    ],
                    "weight": 15
                },
                {
                    "type": "empty",
                    "weight": 85
                }
            ]
        }
    ]
}