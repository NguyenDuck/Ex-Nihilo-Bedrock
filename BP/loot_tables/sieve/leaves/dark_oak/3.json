{
    "pools": [
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "loot_table",
                    "name": "loot_tables/sieve/leaves/dark_oak/2.json"
                }
            ]
        },
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "item",
                    "name": "sapling",
                    "functions": [
                        {
                            "function": "set_data",
                            "data": 5
                        }
                    ],
                    "weight": 15
                },
                {
                    "type": "empty",
                    "weight": 85
                }
            ]
        }
    ]
}