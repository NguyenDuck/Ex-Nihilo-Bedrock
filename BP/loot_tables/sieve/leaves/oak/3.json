{
    "pools": [
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "loot_table",
                    "name": "loot_tables/sieve/leaves/oak/2.json"
                }
            ]
        },
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "item",
                    "name": "sapling",
                    "functions": [
                        {
                            "function": "set_data",
                            "data": 0
                        }
                    ],
                    "weight": 15
                },
                {
                    "type": "empty",
                    "weight": 85
                }
            ]
        }
    ]
}