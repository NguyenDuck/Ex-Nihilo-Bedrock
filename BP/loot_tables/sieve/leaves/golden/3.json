{
    "pools": [
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "loot_table",
                    "name": "loot_tables/sieve/leaves/golden/2.json"
                }
            ]
        },
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "item",
                    "name": "apple",
                    "weight": 15
                },
                {
                    "type": "empty",
                    "weight": 85
                }
            ]
        },
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "item",
                    "name": "golden_apple",
                    "weight": 0.5
                },
                {
                    "type": "empty",
                    "weight": 99.5
                }
            ]
        },
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "item",
                    "name": "exnihilo:silkworm",
                    "weight": 10
                },
                {
                    "type": "empty",
                    "weight": 90
                }
            ]
        }
    ]
}